### ISSUE TITLE
"Annihilate everything" includes the dead sheep that the enemy was gathering food from

### ISSUE THREAD

**Issue by ruben-balea** [Woprok/AOE4-AdvancedGameSettings#67]
It seems that dead sheep become another unit of the player who killed them for food, and of course you can't kill dead sheep again, so the only way to finish the match is to use your villagers to gather food from those dead sheep, it sounds easy but it is very difficult and time consuming to **find** a dead sheep on a giant map.
I don't know if that "annihilate everything" can be tweaked so that it doesn't include the dead sheep.

**Comment by Woprok** [comment]
Interesting, does that happen in non-moded games as well, when you are playing without win conditions ?

Annihilation is basically same as in normal games, when you turn off all win conditions. If it's considering sheeps as unit, then there is something wrong with tags attached to sheep (and Relic should fix that)

**Comment by ruben-balea** [comment]
Well, really I don't know. So far it only happened to me once in a multiplayer game and I assumed it was due to the annihilating everything setting. 
No one knew what else to kill to win until I accidentally clicked on a sheep carcass near a destroyed TC and I found out it still was an unit of one of the enemies, there were deer carcasses there too but those weren't any player's units.

Now I was going to bed, I'm from Spain and it's 0:45 am, but tomorrow I'll try playing a few matches against the AI without mods to see if the AI is able to win if I leave sheep carcasses around my TC.

I will also try to send all my sheep to the enemy TC to see if the AI decides to slaughter them right there, just in case this only happens when the sheep are next to the TC instead of a mill or at random locations.

I will also try find that match, I hope they didn't delete it during the server maintenance they did last week.

**Comment by ruben-balea** [comment]
I tried to watch the replay, one of the players left the game after a couple minutes for no apparent reason, perhaps he lost communication with the server, then about six minutes later the replay threw a synchronization error and stopped.
I imagine that this error was also present in the original match but for some reason the remaining players were all able to continue playing until the end but carrying some corrupted data or the like.

I played a match against the AI using the same map identifier just in case that had anything to do but everything was fine.
